spec/models/generic_file_spec.rb

```diff
@@ -153,7 +153,8 @@
       expect(local[Solrizer.solr_name("mime_type")]).to eq ["image/jpeg"]
       expect(local['all_text_timv']).to eq('abcxyz')
       expect(local[Solrizer.solr_name('belongsToCommunity')]).to eq [community.id]
-      expect(local[Solrizer.solr_name('doi')]).to eq ['doi:10.5072/FKEXAMPLE']
+      expect(local[Solrizer.solr_name('doi', :symbol)]).to eq ['doi:10.5072/FKEXAMPLE']
```

Comment:
I have zero knowledge about Solr, but just curious why we using this `Solrizer.solr_name` helper to access the solr variables? Why not just do `doi_ssim`? Especially if you must have to specify exactly what your looking for (symbol), as it defaults to :stored_searchable (`_tesim`)? Even above on line 154, you can see it doesn't support all the types. It does not support termVectors. 

Also it seems we duplicated majority of our fields in solr? Most of our fields have both a `*_tesim` and a `*_sim` like so:

```
 "language_tesim"=>["Arabic"],
 "language_sim"=>["Arabic"],
```

One for English text stored index multivalued and one for stored (or is this s for string?) index multivalued? Any reason for this?